### PR TITLE
[Nano] Refactor inc according to issue #4228

### PR DESCRIPTION
**Pull request by zhentaocc**
Move all INC related code from `bigdl/nano/quantizatin/neural-compressor` to `bigdl/nano/deps/neural-compressor` .  
Move all INC related test to `test/neural-compresoor` and add extra job to test it.

### CODE REVIEW

**Comment by zhentaocc** [comment]
new inc jenkins job http://10.112.231.51:18888/job/ZOO-PR-nano-inc-test/12/
and other tests http://10.112.231.51:18888/job/ZOO-PR-NanoTests/372

File: python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py [hunk 1]
```diff
@@ -238,12 +238,11 @@ def quantize(self, pl_model: LightningModule,
                 continue
 
         if backend == 'inc':
-            from bigdl.nano.quantization.neural_compressor import QuantizationINC
-            from bigdl.nano.quantization.neural_compressor.pytorch.utils.dataloader import \
-                check_loaders
+            from bigdl.nano.deps.neural_compressor.inc_api import QuantizationINC,\
```
Comment (yangw1234) [inline python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py:241]: move this import to top level?

File: python/nano/src/bigdl/nano/tf/quantization.py [hunk 2]
```diff
@@ -67,9 +67,8 @@ def quantize(self,
      :return:           A TensorflowBaseModel for INC. If there is no model found, return None.
      """
     if backend == 'inc':
-        from bigdl.nano.quantization.neural_compressor import QuantizationINC
-        from neural_compressor.experimental import common
-        from neural_compressor.model.model import TensorflowBaseModel
+        from bigdl.nano.deps.neural_compressor.inc_api import QuantizationINC,\
```
Comment (yangw1234) [inline python/nano/src/bigdl/nano/tf/quantization.py:70]: move this import to top level?

**Comment by yangw1234** [comment]
LGTM

**Comment by yangw1234** [comment]
LGTM

**Pull request merged**